app/models/course.rb

```diff
@@ -323,15 +314,69 @@ class Course < ActiveHash::Base
             'You are an intermediate-level developer',
             'You might be familiar with version control and basic web architecture'
           ]
+        }
+      ]
+    }, {
+      id: 8,
```

Comment:
This is an interesting way around localization; the rest of the app will still be in English, but this class description would be in Portuguese. I'd like to hear what others think about how much localization is feasible for Bridge Troll. cc @anymoto who did the localization for the docs app.